src/chart/SunburstChart.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react';
+import { scaleLinear } from 'victory-vendor/d3-scale';
+import { Surface } from '../container/Surface';
+import { Layer } from '../container/Layer';
+import { Sector } from '../shape/Sector';
+import { Text } from '../component/Text';
+
+export interface SunburstData {
+  name: string;
+  value?: number;
+  fill?: string;
+  children?: SunburstData[];
+}
+
+interface TextOptions {
+  fontFamily?: string;
+  fontWeight?: string;
+  paintOrder?: string;
+  stroke?: string;
+  fill?: string;
+}
+
+export interface SunburstChartProps {
+  data?: SunburstData;
+  width?: number;
+  height?: number;
+  padding?: number;
+  ringPadding?: number;
+  innerRadius?: number;
+  children?: React.ReactNode;
+  fill?: string;
+  stroke?: string;
+  textOptions?: TextOptions;
+}
+
+interface DrawArcOptions {
+  r: number;
+  innerR: number;
+  initialAngle: number;
+  childColor?: string;
+}
+
+const defaultTextProps = {
+  fontFamily: 'sans-serif',
+  fontWeight: 'bold',
+  paintOrder: 'stroke fill',
+  stroke: '#FFF',
+  fill: 'black',
+};
+
+function maxDepth(node: SunburstData): number {
+  if (!node.children || node.children.length === 0) return 1;
+
+  // Calculate depth for each child and find the maximum
+  const childDepths = node.children.map(d => maxDepth(d));
+  return 1 + Math.max(...childDepths);
+}
+
+function polarToCartesian(r: number, angleInDegrees: number): number[] {
+  const angleInRadians = (angleInDegrees * Math.PI) / 180.0;
+  const x = r * Math.cos(angleInRadians);
+  const y = r * Math.sin(angleInRadians);
+  return [x, y];
+}
+
+export const SunburstChart = ({
+  data,
+  children,
+  width,
+  height,
+  padding = 2,
+  ringPadding = 2,
+  innerRadius = 50,
+  fill = '#333',
+  stroke = '#FFF',
+  textOptions = defaultTextProps,
+}: SunburstChartProps) => {
+  // get the max possible radius for a circle inscribed in the chart container
+  const outerRadius = Math.min(width, height) / 2;
+
+  const cx = width / 2,
+    cy = height / 2;
```

Comment:
In other PolarCharts, cy and cx are also overridable. This is totally fine to iterate on, but lets make sure that the props are overall consistent with other PolarCharts.

Reply:
I added new props so now the user can configure startAngle, endAngle, cx, cy, innerRadius and OuterRadius. Added a new story to demonstrate a partial sunburst chart using a custom starting and ending angle. I think that's all the main polar props, but let me know if there are any I missed